dojo-dungeon/src/dojo/rooms/rooms/riddle/Fragen_Pattern.java

```diff
@@ -124,9 +115,10 @@ private BiConsumer<Task, Set<TaskContent>> showAnswersOnHud() {
   }
 
   private Quiz newFreeText() {
+    String fileNamePrefix = "dojo-dungeon/todo-assets/Fragen_Pattern/UML_Klassendiagramm";
```

Comment:
es ist üblich, solche konstanten am anfang der klasse zu definieren, damit man die leichter findet. 

du hattest das schon mal so, aber irgendwie möchtest du das scheinbar nicht?

Reply:
Moin und schönen Sonntag ☕ IntelliJ unterkringelt das, weil die Variable in die Methode eingefügt werden könnte, weil sie innerhalb der Klasse nur einmal gebraucht wird.

Aber ich finde das auch übersichtlicher. Dann ziehe ich sie wieder nach oben.

Reply:
Siehe 046824c